starwars/src/components/CharacterList.jsx

```diff
@@ -0,0 +1,29 @@
+// Takes in Characters and Lists them 
+
+import React from 'react';
+import Character from './Character';
+
+class CharacterList extends React.Component {
+    render () {
+        return (
+            <div className="character-list">
+                {this.props.starwarsChars.map(char => {
+                    return (<Character
+                        key = {char.created}
```

Comment:
Instead of passing them down individually like this, you can pass down the whole `char` obj, then either destructure each of the props that you want to use or use them as `props.whatever` in the component